packages/icons/assets/DaoCardMemberCheck.svg

```diff
@@ -1,4 +1,4 @@
-<svg width="18" height="13" viewBox="0 0 18 13" fill="none" xmlns="http://www.w3.org/2000/svg">
-<path d="M6.5501 13L0.850098 7.29998L2.2751 5.87498L6.5501 10.15L15.7251 0.974976L17.1501 2.39998L6.5501 13Z" fill="#F3F6F8" fill-opacity="0.6"/>
+<svg width="18" height="13" viewBox="0 0 18 13" xmlns="http://www.w3.org/2000/svg">
+<path d="M6.5501 13L0.850098 7.29998L2.2751 5.87498L6.5501 10.15L15.7251 0.974976L17.1501 2.39998L6.5501 13Z"/> 
 </svg>
- 
\ No newline at end of file
+ 
```

Comment:
I think the only change that needs to be made here is changing the `fill` prop to `currentColor` instead of deleting any. That's what I've been doing for other icons and it seems to be working, though now that I think about it, we may want to remove `fill-opacity`. Not sure about that one.